### PR TITLE
modified some language files

### PR DESCRIPTION
**Pull request by Scriptim**

### CODE REVIEW

**Comment by Janmm14** [comment]
I suggest adding a logDebug method with a config file switch wherger to log it actually.

**Comment by Scriptim** [comment]
@Janmm14 why do you not make your own commit?

**Comment by Janmm14** [comment]
@Scriptim because I am currently unable to access my pc, thank my parents.

File: src/de/janhektor/community/tests/TestManager.java [hunk 1]
```diff
@@ -1,5 +1,7 @@
 package de.janhektor.community.tests;
 
+import static de.janhektor.community.utils.Log.*;
```
Comment (Janhektor) [inline src/de/janhektor/community/tests/TestManager.java]: Wildcard-Imports are forbidden by convention.

**Comment by Scriptim** [comment]
@Janhektor u already said that

**Pull request closed without merging**